src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -148,7 +148,8 @@ public static void Rc4AndCngWrappersDontMixTest()
             ecms = new EnvelopedCms();
             ecms.Decode(encodedMessage);
 
-            using (X509Certificate2 cert = Certificates.RSAKeyTransferCapi1.TryGetCertificateWithPrivateKey())
+            // In order to actually use the CAPI version of the key, perphemeral loading must be specified.
```

Comment:
For the existing (netstandard1.3) tests it worked fine.  When I made netcoreapp1.1 tests default to ephemeral it failed (which makes sense, it's precisely what the test is verifying), so I had to force it to stay perphemeral.  So this change is more "keep this test from evolving".